src/lib/mapveu/index.ts

```diff
@@ -1 +1 @@
-export { MapVeuContainer } from './MapVeuContainer';
+export { MapVeuContainer, MapVeuContainer as default } from './MapVeuContainer';
```

Comment:
Does this make these two import statements equivalent?

```js
import { MapVeuContainer } from '<path>'
import MapVeuContainer from '<path>'
```

Reply:
It does, and maybe that's not ideal. I was thinking we would support either, but maybe that's not great?

Reply:
Gotcha! I was just curious. I assumed it was so that we wouldn't have to change as many files.

Reply:
Part of my rationale for the `default` export is so that we don't have to expose the name of the component (I really don't like it). I suppose I could rename the component to something like `MapApp`.